packages/apps/graph/src/graph/query/events.ts

```diff
@@ -69,6 +71,12 @@ builder.queryField('events', (t) =>
             transaction: {
               NOT: [],
             },
+            ...(args.chainId && {
+              chainId: parseInt(args.chainId),
+            }),
+            ...(args.parametersFilter && {
+              parameters: JSON.parse(args.parametersFilter),
+            }),
```

Comment:
The `totalCount` for events does not consider these newly added arguments.
Eg.
![image](https://github.com/kadena-community/kadena.js/assets/112877112/4d619678-b969-4fa1-9a13-4960af5149d7)

Reply:
Good catch, fixed!